avocado/utils/configureNetwork.py

```diff
@@ -0,0 +1,74 @@
+"""
+Configure network when interface name and
+interface IP is available.
+"""
+
+from . import distro
+from . import process
+
+
+def installed_distro():
```

Comment:
I don't see the point of having this function. You can call `distro.dectect().name` directly on the use cases you have.